script/release/DSL-scripts/01.KIE_pushReleaseBranches.sh

```diff
@@ -0,0 +1,81 @@
+# clone the build-bootstrap that contains the other build scripts
+if [ "$SOURCE" == "community-branch" ]; then
+
+   # clone droolsjbm-build-bootstrap branch from droolsjbpm
+   git clone git@github.com:droolsjbpm/droolsjbpm-build-bootstrap.git --branch $BASE_BRANCH
```

Comment:
???
The job has to fetch from somewhere the base branches that will be updated (version) - the resultant branches are the release branches that are finally pushed or to droolsjbpm (community) or jboss-integration (productization).